drivers/usb/udc/Kconfig

```
@@ -14,14 +14,14 @@ if UDC_DRIVER
 config UDC_BUF_COUNT
 	int "Number of buffers in the pool"
 	range 16 256
-	default 16
+	default 100
```

```suggestion
	default 100 if SOC_FAMILY_ENSEMBLE
	default 16
```
This is existing functionality so you should not change old configuration.

---

Done